geomesa-spark/geomesa-spark-jts/src/main/scala/org/locationtech/geomesa/spark/jts/udf/GeometricProcessingFunctions.scala

```diff
@@ -42,23 +40,37 @@ object SQLGeometryProcessingFunctions extends LazyLogging {
   }
 
   val ST_antimeridianSafeGeom: Geometry => Geometry = nullableUDF(geom => {
-    GeohashUtils.getInternationalDateLineSafeGeometry(geom) match {
-      case Success(g) => g
-      case Failure(e) =>
-        logger.warn(s"Error splitting geometry on anti-meridian for $geom", e)
-        geom
+    def degreesToTranslate(x: Double): Double = (((x + 180) / 360.0).floor * -360).toInt
+    if (geom.getEnvelopeInternal.getMinX < -180 || geom.getEnvelopeInternal.getMaxX > 180) {
```

Comment:
i think the logic of this method has changed - is that intentional? the `getInternationalDateLineSafeGeometry` method will split any x span > 180 degrees, now it is only checking for x coordinates outside the 'world'

Reply:
My intention was to keep the logic the same. I thought spatial4j handled splitting the geometry? https://github.com/locationtech/spatial4j/blob/spatial4j-0.6/src/main/java/org/locationtech/spatial4j/shape/jts/JtsGeometry.java#L421

Reply:
ah, you may be right. do we have some unit tests that correspond between both methods to validate they do the same thing?

Reply:
The moved the [unit tests](https://github.com/locationtech/geomesa/pull/1891/files/de805b70d713818aff63d81369e8e18ff2132129#diff-1e74d13f3bee54406d98d9faad0152f3R70) for the UDFs that used to use the old implementation 

There's this unit test for GeoHashUtils. https://github.com/locationtech/geomesa/blob/master/geomesa-utils/src/test/scala/org/locationtech/geomesa/utils/geohash/GeohashUtilsTest.scala#L215
I can double check that it still works here, but I don't think it's worth copying over too?

Reply:
that sounds like a good plan, thanks